src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Figure {
+    private String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    public String infoFromRigure() {
```

Comment:
The method name `infoFromRigure` seems to be a typo. It should likely be `infoFromFigure` to match the naming convention and avoid confusion.